packages/components/src/components/Text/Text.stories.tsx

```diff
@@ -0,0 +1,64 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { View } from 'react-native'
+import React from 'react'
+
+import { Text, TextProps } from './Text'
+import { generateDocs } from '../../utils/storybook'
+
+const meta: Meta<TextProps> = {
+  title: 'Text',
+  component: Text,
+  decorators: [
+    (Story) => (
+      <View
+        style={{
+          flex: 1,
+          justifyContent: 'center',
+          alignItems: 'center',
```

Comment:
Suggestion: add the margin like some of the other more recent stories to give some padding off the edges of the screen for the body variant.

Reply:
Added margin